cluster/openstack-heat/kubernetes-heat/fragments/configure-salt.yaml

```diff
@@ -32,6 +32,10 @@ write_files:
       password=$OS_PASSWORD
       region=$OS_REGION_NAME
       tenant-id=$OS_TENANT_ID
+      [LoadBalancer]
+      lb-version=$LBAAS_VERSION
+      subnet-id=$SUBNET_ID
```

Comment:
What is the current state of the proposal to remove LBaaS V1 from Neutron? I was under the impression that this could happen as early as Newton.